src/utils/utils.ts

```diff
@@ -164,7 +164,7 @@ export function isUserSystemAdmin(actingUser: UserProfile): boolean {
 }
 
 export function isConnected(oauth2user: StoredOauthUserToken): boolean {
-    if (oauth2user && oauth2user.token.access_token && oauth2user.token.access_token !== '') {
+    if (oauth2user?.token?.access_token !== '') {
```

Comment:
In this case `access_token` will be `undefined`, and we will incorrectly assumed it's defined when we assert that it's not the empty string. An empty string is [treated as falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) so the check below will work:

```suggestion
    if (oauth2user?.token?.access_token) {
```

@jfrerich Any reason for the explicit empty string comparison in the existing code?